### DISCUSSION TITLE
Creating a view in csv connection

### DISCUSSION THREAD

**Discussion by der-ada** [dbeaver/dbeaver#8716] [Ideas]
Hi!

I couldn't find any documentation on creating views, especially in the case of a csv connection. Basically, that would be a csv to cvs view. Is this possible?

thx & tia,
ada

**Comment by uslss** [comment]
Driver that is used by DBeaver for now, doesn't support View creation.
By the way, you could try some third-party drivers.

Possibility to create view from UI would be disabled in scope of #9361
